modules/ROOT/pages/penetration-testing-policies.adoc

```diff
@@ -9,40 +9,78 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-If you are deploying applications to xref:index.adoc[Runtime Manager] or APIs on xref:api-manager::index.adoc[API Manager] and your company's security policies require that you run penetration testing on them, there is a way for you to run these. Given that such tests can interfere with other tenants, penetration testing is only allowed on your own workers. It is not allowed for any other services in the Anypoint Platform.
+If your company security policies require that you perform vulnerability and penetration testing 
+against Anypoint Platform, request a Security Assessment before proceeding.
 
-External vulnerability scans, via an Approved Scanning Vendor (ASV), as described in section 11.2.2 of the PCI standard may be performed at will and do not require prior approval. Penetration testing as described by section 11.3.1 of the PCI standard does require prior approval from MuleSoft.
 
-CloudHub Workers, by default, do not use static IPs and cannot be properly tested as their IPs may change. Please enable static IPs, as described in xref:deploying-to-cloudhub.adoc#static-ips[Static IPs].
+== Request a Security Assessment 
 
-To perform such testing, you must provide a written request 5 business days in advance to security@mulesoft.com along with:
+To request a Security Assessment:
 
-* Start and end time of testing.
-* Hosts to include in the test.
-* Source IP address or range.
-* Contact information, including a phone number and email, for the individual or team conducting the test.
-* You agree to only audit CloudHub workers, and not audit any other platform services which are common to all tenants.
-* You agree not to audit shared workers.
-* Send an email to security@mulesoft.com upon start and completion of tests.
-* Notify security@mulesoft.com of any vulnerabilities found immediately.
-* You may only audit once a quarter.
+. Contact your Salesforce Account team to request the Security Assessment Request form and Security Assessment Agreement.
```

Comment:
Here the init. caps. make more sense: the titles of the forms.
I'd think twice about Account team, however, even with Salesforce before it. 
Meh....